plugins/pencilblue/controllers/salesforce/change_password.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = function SalesforceChangePasswordModule(pb) {
+    const SalesforceAPIController = require('../api/salesforce/salesforce_api')(pb);
```

Comment:
We need a better way of doing... this... in general in the future.  Typically we attach stuff like this to pb (at least in the core PB Repo we do.) . so that is an option.  Not going to block the PR on it though.